lib/ember-cli/helpers.rb

```diff
@@ -25,10 +25,18 @@ def non_production?
       !Rails.env.production? && Rails.configuration.consider_all_requests_local
     end
 
+    def inject_install_dependencies_task!
```

Comment:
Could we move the definition if this task into a separate file? Kind if like the way it's described [here](http://blog.nathanhumbert.com/2010/02/rails-3-loading-rake-tasks-from-gem.html)